src/contentscripts/messages/create.js

```diff
@@ -0,0 +1,35 @@
+var inject = require('../../../lib/inject.js');
+var $ = require('jquery');
+var xhttp = new XMLHttpRequest();
+var DOM = require('jsx-dom-factory');
+
+
+xhttp.onreadystatechange = function() {
+	if (xhttp.readyState == 4 && xhttp.status == 200) {
+		var result = xhttp.responseText;
+		var replaced = result.replace("CreateMessageViewModel,f,t,i,r,u;","CreateMessageViewModel,f,t,i,r,u;msgsystem = n;");
```

Comment:
awesome, I was going to suggest this approach. I also thought we could use a patch file but this is simpler.